randovania/data/json_data/prime1.txt

```diff
@@ -3032,9 +3069,7 @@ Asset id: 4272124642
   > Dock to Save Station Mines B
       After Security Drone Defeated
   > Event - Security Drone
-      Any of the following:
-          X-Ray Visor
-          Combat (Beginner) and Invisible Objects (Beginner)
+      Energy Tank or Combat (Beginner)
 
 > Dock to Quarantine Access A; Heals? False
   * Ice Door to Quarantine Access A/Dock to Central Dynamo
```

Comment:
Should update the requirements below for the drone event to match the requirements above, with added pbs of course and the methods to reach the event in the first place

Reply:
I've added a drone fight trigger node to central dynamo to simplify the room. There should only be one connection to climb the room and one connection to fight the drone now.

Reply:
Also, what do you think about the difficulty of the HBJ(?) to escape Central Dynamo? I can't actually do this trick so I took a guess at how hard it is. Same for the 21% Vent Shaft half pipe bomb jump

Reply:
Hmm to me it looks like the old paths for the drone still exist, with "Xray or invisible objects." Unless thats been changed outside the PR already.

As for the bomb jumps, I think that is fair for now.

Reply:
e6d536da should have removed all of the old paths to the drone, unless I missed something